Vagrantfile

```diff
@@ -0,0 +1,26 @@
+Vagrant.configure(2) do |config|
+    config.vm.box = "fedora/25-cloud-base"
+    config.vm.network "forwarded_port", guest: 9090, host: 9090
+
+    config.vm.synced_folder ".", "/vagrant", disabled: true
+    config.vm.synced_folder "dist/", "/usr/local/share/cockpit/" + File.basename(Dir.pwd)
```

Comment:
Indeed, that makes it easier to figure out when the module doesn't show up in the VM. Thanks for the suggestion! Pushed another commit that adds the message and creates the directory when it doesn't exist.